### PR TITLE
Change behaviour of default opa policy

### LINKED ISSUE #190: Kubeflow sample fails with error
Kubeflow sample fails with error running on Kind cluster:

Status:
  Conditions:
    Message:  An error was received for asset {"ServerName":"cocoMDS3","AssetGuid":"b47268c3-9f69-41f9-84c3-3818f424407d"} . Error description: Copy of the data is required but can not be done according to the governance policies.

Here is how I call create_new_asset.sh:
./create_new_asset.sh ../../../samples/kubeflow/example_transactions.csv.json 'finance'

in  opa-connector log the required action is encrypted which is not supported:

2020/12/08 07:28:55 usedPolicies:  [description:"test for transactions dataset that encrypts some columns by name"  description:"test for transactions dataset that encrypts some columns by name" ]
2020/12/08 07:28:55 Received evaluation : {
        "dataset_decisions": [
                {
                        "dataset": {
                                "dataset_id": "{\"ServerName\":\"cocoMDS3\",\"AssetGuid\":\"b47268c3-9f69-41f9-84c3-3818f424407d\"}"
                        },
                        "decisions": [
                                {
                                        "operation": {
                                                "type": 1,
                                                "destination": "US"
                                        },
                                        "enforcement_actions": [
                                                {
                                                        "name": "encrypted",
                                                        "id": "encrypted-ID",
                                                        "level": 2,
                                                        "args": {
                                                                "column_name": "nameDest"
                                                        }
                                                },
                                                {
                                                        "name": "encrypted",
                                                        "id": "encrypted-ID",
                                                        "level": 2,
                                                        "args": {
                                                                "column_name": "nameOrig"
                                                        }
                                                }
                                        ],
                                        "used_policies": [
                                                {
                                                        "description": "test for transactions dataset that encrypts some columns by name"
                                                },
                                                {
                                                        "description": "test for transactions dataset that encrypts some columns by name"
                                                }
                                        ]
                                }
                        ]

### PR DESCRIPTION
**Pull request by rohithdv**
This PR fixes #190 . 
The cause of this error was that the default behaviour of the example opa policy is set to return Encrypt enforcement action instead of Redact enforcement action. This is now being changed in this PR. Also there were some spelling errors for redact. This is also corrected.

### CODE REVIEW

File: third_party/opa/data-and-policies/user_policies.rego [hunk 1]
```diff
@@ -42,12 +42,14 @@ transform[action] {
     not dp.check_destination([dp.GeoDestinations.US])
     
     column_names := dp.column_with_any_tag(["SPI", "SMI"])
-    action = dp.build_encrypt_column_action(column_names[_], dp.build_policy_from_description(description))
+    #action = dp.build_encrypt_column_action(column_names[_], dp.build_policy_from_description(description))
+    action = dp.build_redact_column_action(column_names[_], dp.build_policy_from_description(description))
```
Comment (shlomitk1) [inline third_party/opa/data-and-policies/user_policies.rego:46]: This policy is not related to the example (with tag = Finance). Why is it changed?

Comment (rohithdv) [reply third_party/opa/data-and-policies/user_policies.rego:46]: Since we don't support encrypt enf action yet I have made redact action as the default transformation. Please let me know if this is ok. Thanks.

Comment (shlomitk1) [reply third_party/opa/data-and-policies/user_policies.rego:46]: OK.

**Review by shlomitk1 — APPROVED** [review]

**Review by revit13 — APPROVED** [review]
